setup.py

```diff
@@ -302,14 +302,14 @@ def get_long_description():
     extras_require={
         "docs": [
             "enthought-sphinx-theme",
-            "Sphinx",
+            "Sphinx==2.3.1",
         ],
         "test": [
             "Cython",
             "flake8",
             "mypy",
             "setuptools",
-            "Sphinx>=2.1.0",
+            "Sphinx==2.3.1",
```

Comment:
I think I was the one who suggested downgrading sphinx, but I think we don't need to go all the way back to `2.3.1`, e.g. the last build with sphinx 3.1.2 does work fine.

And to stay current with less effort, do you think we could do `"Sphinx!=3.2.0"` instead? Then when there is a bug fix release, we will be able to install that. Same for the EDM dependency.

Reply:
(EDM does not have 3.2.0 yet, but it won't mind a `!=3.2.0` as long as it can be satisfied.)